src/views/Viewer.vue

```diff
@@ -968,6 +969,24 @@ export default {
 			this.trapElements = []
 		},
 
+		// Update etag of updated file to break cache.
+		async handleFileUpdated({ fileid }) {
+			const index = this.fileList.findIndex(({ fileid: currentFileId }) => currentFileId === fileid)
+			const file = this.fileList[index]
+
+			if (file === undefined) {
+				return
+			}
+
+			const filename = file.source.replace(new RegExp(`^${getRootPath()}`), '')
+			const fetchedFile = await getFileInfo(filename)
```

Comment:
this is a huge request just to get an etag :/

Reply:
Is that a thing already in place?

Reply:
Kind of! We just never have an updated event fired from files for now.
Currently only Viewer fires `files:node:updated`, but we're not using the File/Folder objects here yet